assets/js/state/selectors/activityLogsSettings.test.js

```diff
@@ -0,0 +1,118 @@
+import {
+  getActivityLogsSettings,
+  getActivityLogsSettingsLoading,
+  getActivityLogsSettingsSaved,
+} from './activityLogsSettings';
+
+describe('Activity Logs Settings selector', () => {
+  describe('get activity logs settings', () => {
+    const stateScenarios = [
+      {
+        loading: false,
+        settings: {
+          retention_time: { value: 2, unit: 'week' },
+        },
+        errors: null,
+        editing: false,
+      },
+      {
+        loading: true,
+        settings: {
+          retention_time: undefined,
+        },
+        errors: null,
+        editing: false,
+        testingConnection: false,
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/trento-project/web/pull/2727/commits/e2f68277aea2579e811502f8bde632dec4692e82